pkg/rc/replication_controller.go

```diff
@@ -196,6 +197,11 @@ func (rc *replicationController) WatchDesires(quit <-chan struct{}) <-chan error
 	errOutChannel := make(chan error)
 	channelsClosed := make(chan struct{})
 
+	// get random wait time to check for missing artifacts bewteen 1 - 60 minutes
+	randomTime := rand.Intn(60) + 1
+	shouldInitialCheck := time.After(fmt.Sprintf("%dm", randomTime))
```

Comment:
Does `time.After()` take a string? I think you probably want `randomTime * time.Minute` here